src/store/actions/index.tsx

```diff
@@ -0,0 +1,20 @@
+import {
+  Test,
+  ActionTypes,
+  TEST_FIRST_DISPATCH,
+  TEST_SEC_DISPATCH,
+} from '../types';
+
+export const firstTestDispatch = (resp: Test): ActionTypes => {
+  return {
```

Comment:
I think better will be used response or res instead of resp.
TO GET ALONG

Reply:
We can change it later, however it's not a error because resp is a abbreviation of respond